java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -4034,6 +4074,16 @@ private static native long stringReplaceWithBackrefs(long columnView, String pat
    */
   private static native long containsRe(long cudfViewHandle, String pattern) throws CudfException;
 
+  /**
+   * Native method for checking if strings match the passed in like pattern
+   * and escape character.
+   * @param cudfViewHandle native handle of the cudf::column_view being operated on.
+   * @param pattern string like pattern.
+   * @param escapeChar string escape character.
+   * @return native handle of the resulting cudf column containing the boolean results.
+   */
+  private static native long like(long cudfViewHandle, long pattern, long escapeChar) throws CudfException;
```

Comment:
Essentially all 3 parameters are kind of memory handle. So we should better name them more consistently and meaningfully. For example: `columnHandle`, `patternHandle`, `escapeCharHandle` or something similar.